### PR TITLE
updates

### PR DESCRIPTION
**Pull request by roninsightrx**
Went through all vignettes:
- made some textual updates + code updates
- made sure that all code works, and is formatted nicely
- Consolidated absorption model examples into single vignette
- changed nlmixr to nlmixr2
- made sure R output is shown truncated (head)
- misc other things

### CODE REVIEW

File: vignettes/estimation-nlmixr.Rmd
Comment (karawoo) [inline vignettes/estimation-nlmixr.Rmd:13]:
```suggestion
a convenient translator to be able to use the [`nlmixr2` R package](https://nlmixr2.github.io/nlmixr2/) for parameter estimation. `nlmixr2` allows fitting of
```

File: vignettes/model.Rmd
Comment (roninsightrx) [reply vignettes/model.Rmd:113]: The intention is indeed for it to be `F1`. Explicitly adding the parameters will make the model compile
```
   parameters = list(KA = 1, CL = 5, V = 50, F1 = 0.7)
```

**Review by karawoo — APPROVED** [review]
lgtm, I can fix that one example after we merge this to the RXR-1050 branch, looks like there's still a little I need to do to get the build passing too.

**Pull request merged**
